### ISSUE TITLE
[UX] Interface to add a contact is hard to find / misleading

### ISSUE THREAD

**Issue by bohwaz** [oxen-io/session-android#1192]
**Code of conduct**

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

After trying the app with multiple persons, the main issue we found is that people are lost, there is no "Add contact" link or text in the UI, going through "New message" is unclear.

Suggestion : when clicking the "+" green button, the "New message" item should be renamed "Add new contact".

**Comment by KeeJef** [comment]
Yeah i think this valid, I've earmarked this and we will be looking at this in our upcoming onboarding project

**Comment by KeeJef** [comment]
After consideration and internal discussions, we've decided to stick with the 'New Message' menu button for now. Definitely  'Add Contact' is a common term in many chat applications, but during our user testing, we didn't observe substantial confusion about the 'New Message' button. It also seems to align better with the intended functionality, since when you click this button you begin the process of sending a new message request to the user, rather than simply adding an empty contact to be messaged at a later time. Closing for now
